public/app/core/components/AppChrome/TopBar/HelpTopBarButton.tsx

```diff
@@ -34,7 +34,12 @@ export const HelpTopBarButton = memo(function HelpTopBarButton({ isSmallScreen }
   if (isSmallScreen || !enrichedHelpNode.hideFromTabs || interactiveLearningPluginId === undefined) {
     return (
       <Dropdown overlay={() => <TopNavBarMenu node={enrichedHelpNode} />} placement="bottom-end">
-        <ToolbarButton iconOnly icon="question-circle" aria-label={t('navigation.help.aria-label', 'Help')} />
+        <ToolbarButton
+          iconOnly
+          icon="question-circle"
+          aria-label={t('navigation.help.aria-label', 'Help')}
+          tooltip={t('navigation.help.tooltip', 'Help and documentation')}
```

Comment:
I'm not sure if we have product design guidelines for tooltips but I wonder if this is more accurate in description:

```suggestion
          tooltip={t('navigation.help.tooltip', 'Get help and useful links')}
```